### ISSUE TITLE
Play: Unable to play the public .mp3 URL 

### ISSUE THREAD

**Issue by Shahz-B** [jambonz/jambonz-feature-server#537]
We have encountered a recurring issue that we previously addressed. @davehorton had investigated and resolved this issue in the past. He identified that the rejection was linked to Freeswitch's use of an HTTP HEAD request for file size retrieval.

The issue has resurfaced, and over the last 2-3 days, we've observed the same problem occurring again. here’s one of the callSid forreference: `0dc0e634-8b99-40e4-b189-e8b86cc43fbd`

**Comment by davehorton** [comment]
Yes, there is an issue in freeswitch where it tries to send a HEAD request, which in this case is rejected.  This requires a patch to freeswitch
